h2o-py/tests/testdir_algos/xgboost/pyunit_xgboost_eval_metric_stopping.py

```diff
@@ -35,8 +36,25 @@ def test_eval_metric_early_stopping():
 
     assert_same_scoring_history(model_actual, model_expected, "training_custom", "training_mae")
 
+    return model_actual.scoring_history()
+
+
+def test_eval_metric_early_stopping():
+    check_eval_metric_early_stopping()
+
+
+def test_eval_metric_early_stopping_native_scoring_only():
+    history = check_eval_metric_early_stopping(True)
+    for col in history.columns:
+        if col.startswith("training_") and col != "training_custom":
+            is_missing = history[col].isnull()
+            # scoring history is not defined for H2O metrics...
+            assert is_missing.sum() == len(is_missing) - 1
+            # ...except for the final scoring metric
+            assert not is_missing.iloc[-1]
+
 
-if __name__ == "__main__":
-    pyunit_utils.standalone_test(test_eval_metric_early_stopping)
-else:
-    test_eval_metric_early_stopping()
+pyunit_utils.run_tests([
+    test_eval_metric_early_stopping,
+    test_eval_metric_early_stopping_native_scoring_only
+])
```

Comment:
Why this test file doesn't follow the 
`if __name__ == "__main__": 
...
else:
...
`
structure as all other test files in this folder do?

Reply:
`run_tests` was added later than most of these tests, it is a newer mechanism and is better when you have multiple test cases in the same file. Eg. StackEnsemble tests almost exclusively use run_tests. There is not loss of functionality.

Thanks for the comment, good to check!